src/screens/Matches.tsx

```diff
@@ -1,23 +1,95 @@
 import * as shape from 'd3-shape';
 import React from 'react';
-import {Image, SafeAreaView, StyleSheet, Button, View} from 'react-native';
+import {Image, SafeAreaView, StyleSheet, Button, View, Dimensions, ScrollView } from 'react-native';
 
 import {Line} from 'react-native-svg';
 import {LineChart, Path} from 'react-native-svg-charts';
 
-import {Block, Text} from '../components';
-import {CardList} from '../components/matches';
+import {Block, Text, Header, Card} from '../components';
+import {default as CardList, CardListProps} from '../components/matches/CardList';
```

Comment:
You don't need CardListProps replace with CardList['props']

Reply:
yoink